back-end/src/server.ts

```diff
@@ -17,5 +17,4 @@ route.get('/', (req: Request, res: Response) => {
 
 app.use(route)
 
-
-app.listen(3001, () => 'server running on port 3001')
\ No newline at end of file
+app.listen(process.env.PORT ?? 3000, () => 'server running on port 3001')
```

Comment:
I think that would be better if the port on the message matches the `process.env.PORT`